examples/mobile/HomeApp/js/app.js

```diff
@@ -369,7 +391,24 @@ import Storage from "./clipping-storage.js";
 			});
 		});
 
-		// activate / deactivate mini control cards
+		// set proper order of cards
+		// @todo change inline styles to css class after merge HomeApp branches
+		webclipsContainer.style.display = "flex";
+		webclipsContainer.style.flexDirection = "column";
+		currentWebClipsCards = [].slice.call(webclipsContainer.querySelectorAll(".ui-card[data-url],.ui-card[data-src]"));
```

Comment:
Just I have a question.
I know about the "data-src" of ui-card. It's or card resource path like "webclip/apps-on-tv".
What's for use "data-url"?

Reply:
Hi,
Cards may be in two states one before load of card content (`data-src`) and after load of content (`data-url`). We need check both attributes.

Reply:
ok. I see. Thank you for your kind explanation.